src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.html

```diff
@@ -22,18 +22,32 @@
     </span>
   </div>
   <div class="card-body fill-plain">
-    <div>
+    <form #courseForm="ngForm" (ngSubmit)="onSaveCourse()">
       <div class="form-row form-group">
         <label class="col-sm-3">Course ID:</label>
         <div class="col-sm-3">
-          <input class="col-xs-12 form-control" maxlength="40" tabindex="1" [(ngModel)]="course.courseId" disabled>
+          <input class="col-xs-12 form-control" type="text" tabindex="1"
+                 #courseId="ngModel" [(ngModel)]="course.courseId" name="courseId"
+                 [maxlength]="FormValidator.COURSE_ID_MAX_LENGTH" required disabled>
+          <div [hidden]="courseId.valid || (courseId.pristine && courseId.untouched)"
+               class="invalid-field">
+            <i class="fa fa-exclamation-circle" aria-hidden="true"></i>
+            The field Course ID should not be empty.
```

Comment:
Can this ever appear? It looks like the course ID cannot be edited.

Reply:
In this form no, I decided to add it for future-proofing in the event #9976 gets addressed. If that's not necessary however I can remove it :)

Reply:
Hmm, let's leave it out for now. If someone wants to write a PR to solve the issue they can refer to this referenced PR in the issue. We want to try to reduce unused code so maintainability is easier.